### ISSUE TITLE
Remove old Woven diagrams

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#4189]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)| @thijstriemstra reported|
|-|-|
|Trac ID|trac#4189|
|Type|defect|
|Created|2009-12-25 17:44:49Z|
|Branch|https://github.com/twisted/twisted/tree/woven-png-4189|

See the ```woven_*.png``` images in [doc/web/img](doc/web/img).

<details><summary>Searchable metadata</summary>

```
trac-id__4189 4189
type__defect defect
reporter__thijs thijs
priority__normal normal
milestone__ 
branch__branches_woven_png_4189 branches/woven-png-4189
branch_author__thijs thijs
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation documentation
time__1261763089000000 1261763089000000
changetime__1263504207000000 1263504207000000
version__None None
owner__ 
cc__thijs cc__jesstess
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_1"></a>|@thijstriemstra commented|
|-|-|

(In [27879]) Branching to 'woven-png-4189'

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_2"></a>|@thijstriemstra commented|
|-|-|

(In [27880]) Remove old woven diagrams that are not being used, refs [#4189](https://github.com/twisted/twisted/issues/4189)

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)|@thijstriemstra removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_4"></a>|@exarkun set owner to @thijstriemstra|
|-|-|

There are a few more woven-related files in the docs.  Probably everything in `doc/core/howto/listings/HelloWorld` can go too?

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_5"></a>|@thijstriemstra commented|
|-|-|

(In [28000]) Remove old woven helloworld, refs [#4189](https://github.com/twisted/twisted/issues/4189)

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_6"></a>|@thijstriemstra removed owner|
|-|-|

I grepped the trunk and those hello world examples were the only relevant files with a reference to woven. Branch up for review.

**Comment by twisted-trac** [comment]
|[<img alt="jesstess's avatar" src="https://avatars.githubusercontent.com/u/188336?s=50" width="50" height="50">](https://github.com/jesstess)|@jesstess set owner to @jesstess|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="jesstess's avatar" src="https://avatars.githubusercontent.com/u/188336?s=50" width="50" height="50">](https://github.com/jesstess)<a name="note_8"></a>|@jesstess set owner to @thijstriemstra|
|-|-|

Looks good. Re-grepped and can confirm no more references to woven; everything still lores fine. Please merge!

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_9"></a>|@thijstriemstra commented|
|-|-|

(In [28003]) Add NEWS file, refs [#4189](https://github.com/twisted/twisted/issues/4189)

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_10"></a>|@thijstriemstra set status to `closed`|
|-|-|

(In [28004]) Merge woven-png-4189: Remove last references to twisted.web.woven from the documentation.

Author: thijs
Reviewer: exarkun, jesstess
Fixes: [#4189](https://github.com/twisted/twisted/issues/4189)

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
